app/helpers/bid_placeholder_helper.rb

```diff
@@ -0,0 +1,11 @@
+module BidPlaceholderHelper
+  def bid_placeholder(item)
+    if item.top_bid.present?
+      number_to_currency(item.top_bid.amount).gsub("$", "")
+    elsif item.starting_bid.present?
+      number_to_currency(item.starting_bid).gsub("$", "")
```

Comment:
Look at the number_to_currency api.  You can prob tell it to skip the "$" instead of removing it by hand.  http://api.rubyonrails.org/classes/ActiveSupport/NumberHelper.html#method-i-number_to_currency